metrics/details/init.lua

```diff
@@ -29,6 +29,15 @@ function Registry:is_registered(collector)
     return false
 end
 
+function Registry:get_registered(collector)
+    for _, c in ipairs(self.collectors) do
+        if c.name == collector.name and c.kind == collector.kind then
```

Comment:
if name and kind inconsistence make an assert

Reply:
I.e. if one of them is nil?

Reply:
added assertions